app/models/accredited_individual.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'accredited_representation/constants'
+
+class AccreditedIndividual < ApplicationRecord
+  # rubocop:disable Rails/HasAndBelongsToMany
+  has_and_belongs_to_many :accredited_organizations,
+                          class: 'AccreditedOrganization',
+                          join_table: 'accredited_individuals_accredited_organizations'
+
+  # rubocop:enable Rails/HasAndBelongsToMany
```

Comment:
Instead of disabling this cop, can you modify the code to be compliant? https://www.rubydoc.info/gems/rubocop/0.58.1/RuboCop/Cop/Rails/HasAndBelongsToMany

Reply:
There's no reason for this to be `has_many :through`. I think this can stay as-is.

Reply:
seems like `has_and_belongs_to_many` doesn't offer the same flexibility, but if this works for your schema, 👍